lib/post/widgets/comment_card.dart

```diff
@@ -22,6 +22,7 @@ class CommentCard extends StatefulWidget {
   final Function(int, bool) onCollapseCommentChange;
   final Function(int, bool) onDeleteAction;
   final Function(CommentView, bool) onReplyEditAction;
+  final Function(int, bool) onReportAction;
```

Comment:
Any reason for the `bool` param? It looks like it's ignored.

Reply:
Yes it is. Had some concerns  on 

1. If we should hide the reported comment.
2. What happens on reload.

From what I have seen Lemmy marks it as reported and no action in the UI.

Reply:
That's a fair question. Are you suggesting that we might want to internally track user-reported comments and not show them? What happens in the web UI?

I'm still confused about the usage of this param. The `onReportAction` just invokes the new dialog. Unlike something like upvote or save, there's no opposite action to perform; it's always just reporting, so I'm still not grasping why we need the bool. I'm sure I'm missing something though. 😊

Reply:
Removed the bool. 
 Assuming we were hiding the comment on report , we could use this to remove the comment.